plugin/pkg/scheduler/api/types.go

```diff
@@ -165,7 +165,7 @@ func (h HostPriorityList) Less(i, j int) bool {
 	if h[i].Score == h[j].Score {
 		return h[i].Host < h[j].Host
 	}
-	return h[i].Score < h[j].Score
+	return h[i].Score > h[j].Score
```

Comment:
It looks like the more standard way to do this is to leave it as < and then do the sort using 
`sort.Sort(sort.Reverse(priorityList))`